web/src/helpers/stringToHTML.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import dompurify from 'dompurify';
+
+export const stringToHTML = (str: string) => {
+	const sanitizer = dompurify.sanitize;
+	
+	return <div dangerouslySetInnerHTML={{__html: sanitizer(str)}}></div>;

```

Comment:
https://www.npmjs.com/package/html-react-parser

Please, check this
Maybe better we can use this library for transforming string to html